ceph_iscsi_config/common.py

```diff
@@ -641,6 +641,29 @@ def _commit_rbd(self, post_action):
     def commit(self, post_action='close'):
         self._commit_rbd(post_action)
 
+    def normalise_controls(self):
+        changed = False
+        for section in ('targets', 'disks', 'gateways'):
```

Comment:
Do we really need `gateways` here? I think we only store `controls` in `/target/<iqn>/controls` and `/disks/<disk_id>/controls`.

Reply:
Probably not. Originally I just wanted to loop through all main settings, but then realised everything we needed to change were in controls. But wasn't sure if gateways got them so just left it in.

I'll double check the code and remove `gateways` if not required.

Reply:
Looking at the gwcli code, we don't seem to ever set controls in the gateways, so will remove it.